### ISSUE TITLE
Colorbar's maximum bins is fixed to 10

### ISSUE THREAD

**Issue by GonzaloSaez** [lukelbd/proplot#239]
### Description

In all the examples available for this package and in my tests, the colorbar maximum number of bins is fixed to 10. There is doesn't seem to by any to changing this

### Steps to reproduce

A "[Minimal, Complete and Verifiable Example](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports)" will make it much easier for maintainers to help you.

```python
fig, ax = proplot.subplots()
pc = plt.imshow(p.image, extent=[file.bounds[0], file.bounds[1], file.bounds[2], file.bounds[3]], cmap=my_cmap, vmin=0.5, vmax=1.5)
plt.colorbar(pc, ax=ax)
ax.set_xlim([0, 1])
ax.set_ylim([0, 1])
ax.set_xlabel(r"$x/R$")
ax.set_ylabel(r"$y/R$")
plt.show()
```

**Actual behavior**: [What actually happened]
![image](https://user-images.githubusercontent.com/11050889/102356830-fe1ae680-3fad-11eb-925b-372c5188970f.png)

### Equivalent steps in matplotlib

Please make sure this bug is related to a specific proplot feature. If you're not sure, try to replicate it with the [native matplotlib API](https://matplotlib.org/3.1.1/api/index.html). Matplotlib bugs belong on the [matplotlib github page](https://github.com/matplotlib/matplotlib).

```python
fig, ax = plt.subplots()
pc = plt.imshow(p.image, extent=[file.bounds[0], file.bounds[1], file.bounds[2], file.bounds[3]], cmap=my_cmap, vmin=0.5, vmax=1.5)
plt.colorbar(pc, ax=ax)
ax.set_xlim([0, 1])
ax.set_ylim([0, 1])
ax.set_xlabel(r"$x/R$")
ax.set_ylabel(r"$y/R$")
plt.show()
```

![image](https://user-images.githubusercontent.com/11050889/102357071-40dcbe80-3fae-11eb-8919-06f7c30fed9d.png)

**Comment by pratiman-91** [comment]
I think you are looking for `levels`.
Try this:
```python
fig, ax = proplot.subplots()
pc = plt.imshow(p.image, extent=[file.bounds[0], file.bounds[1], file.bounds[2], file.bounds[3]], cmap=my_cmap, vmin=0.5, vmax=1.5, levels=30)
plt.colorbar(pc, ax=ax)
ax.set_xlim([0, 1])
ax.set_ylim([0, 1])
ax.set_xlabel(r"$x/R$")
ax.set_ylabel(r"$y/R$")
plt.show()
```

**Comment by GonzaloSaez** [comment]
> I think you are looking for `levels`.
> Try this:
> 
> ```python
> fig, ax = proplot.subplots()
> pc = plt.imshow(p.image, extent=[file.bounds[0], file.bounds[1], file.bounds[2], file.bounds[3]], cmap=my_cmap, vmin=0.5, vmax=1.5, levels=30)
> plt.colorbar(pc, ax=ax)
> ax.set_xlim([0, 1])
> ax.set_ylim([0, 1])
> ax.set_xlabel(r"$x/R$")
> ax.set_ylabel(r"$y/R$")
> plt.show()
> ```

It worked! Thank you so much!

**Comment by scottstanie** [comment]
Is there a way to change to a higher default `levels`, so that you don't need to specify something like `levels=50` to match matplotlib's `imshow`?

**Comment by pratiman-91** [comment]
Maybe you want to look at this section of the documentation.
[https://proplot.readthedocs.io/en/latest/api/proplot.axes.cmap_changer.html?highlight=imshow#proplot.axes.cmap_changer](url)

**Comment by scottstanie** [comment]
Thanks @pratiman-91 , but I'm not sure I follow- this the `cmap_changer` something you can call at the top of the script to set the `levels`? I'm only seeing that it wraps `imshow` so that you can pass `levels` as a keyword arg.

I'm seeing in https://proplot.readthedocs.io/en/latest/2dplots.html#Discrete-colormap-levels that the decision was made to default to a Discrete colormap; my question was whether there's a simple way to say once "i'd like the non-discrete default", like with `rcparams` or somthing

**Comment by pratiman-91** [comment]
This might be help:
`Default is rc[‘image.levels’] = 11.`
You can change here.

**Comment by lukelbd** [comment]
Yep this is more-or-less a duplicate of #233. Solution: Use `N=big_number` or `levels=big_number`, or do this globally using `rc['image.levels'] = big_number`.

In a future version I might add an exception for `imshow` where the default number of levels is much larger. It could also be useful to optionally disable `DiscreteNorm`, but don't think that has much of a functional difference from this approach of using `DiscreteNorm` with a large number of levels.

**Comment by lukelbd** [comment]
FYI I decided that I will indeed disable this feature for `imshow` and `matshow` plots by default, and I'll add an `rc['image.boundarynorm']` option that can be used to disable `DiscreteNorm` proplot-wide. I think there is an advantage to being able to opt out of `DiscreteNorm` since it probably slows down drawing quite a bit for large numbers of levels.

See #233 for updates.

**Comment by lukelbd** [comment]
This is now fixed! See #233
